.github/workflows/docker.yml

```diff
@@ -34,5 +49,6 @@ jobs:
         context: .
         file: ./docker/ci/helper.Dockerfile
         push: true
-        tags: ghcr.io/${{ github.repository }}/ipa-helper:latest
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
```

Comment:
Does it work when `labels` section is not defined (also outputs seems to be implicit)

Reply:
It ran fine for me here: https://github.com/cberkhoff/ipa/actions/runs/10744939741/job/29802940811